VideoPlusChat/src/main/java/com/microsoft/device/display/samples/videochatcomposesample/ui/views/MainPage.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.display.samples.videochatcomposesample.ui.views
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.focus.FocusManager
+import androidx.compose.ui.platform.LocalFocusManager
+import com.microsoft.device.display.samples.videochatcomposesample.models.InfoProvider
+import com.google.android.exoplayer2.ExoPlayer
+import com.microsoft.device.dualscreen.twopanelayout.TwoPaneLayout
+import com.microsoft.device.dualscreen.windowstate.WindowState
+
+var infoProvider: InfoProvider = InfoProvider()
```

Comment:
I think it would be better for `infoProvider` to become a dependency injected into the Composable, like `windowState` and `player`.

Reply:
yeah youre totally right, I was thinking it was going to be used deeper in the composables but its only used at the top level